package.json

```diff
@@ -79,6 +79,7 @@
   "dependencies": {
     "@types/dom-mediacapture-record": "^1.0.7",
     "@types/ua-parser-js": "^0.7.35",
+    "aws-embedded-metrics": "^2.0.4",
```

Comment:
This dependency doesn't seem to be related to this PR?

Reply:
Hmm, thats weird. Have removed it.